### PR TITLE
Fix/521 override dialog styles

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes [521](https://github.com/etn-ccis/blui-react-workflows/issues/521) .

#### Changes proposed in this Pull Request:

- Add `sx` prop to dialogConfig
- Update changelog
-

#### Screenshots / Screen Recording (if applicable)

-
Screenshot 2023-12-13 at 6 15 48 PM

#### To Test:

- Throw an error from login auth UI actions
- Add the below code in `login-workflow/example/src/screens/Login.tsx`

`
<LoginScreen
        projectImage={logo}
        header={<DebugComponent />}
        errorDisplayConfig={{
            mode: 'dialog',
            messageBoxConfig: {
                dismissible: true,
                position: 'top',
                sx: { backgroundColor: 'green' },
            },
            dialogConfig: {
                title: 'Change Title Color',
                sx: {
                    '& .MuiTypography-h6': {
                        color: 'red',
                    },
                },
            },
        }}
    />
`

#### Any specific feedback you are looking for?

- Sorry for unformatted code.

### CODE REVIEW

**Review by JeffGreiner-eaton — APPROVED** [review]
look good. not sure if this is a fixed in change log since it wasn't a requirement from the initial implementation.

**Comment by ektaghag-eaton** [comment]
> look good. not sure if this is a fixed in change log since it wasn't a requirement from the initial implementation.

Sure jeff will remove it.

File: login-workflow/src/components/Error/ErrorManager.tsx [hunk 1]
```diff
@@ -36,6 +36,7 @@ export type ErrorManagerProps = {
     dialogConfig?: {
         dismissLabel?: string;
         title?: string;
+        sx?: SxProps;
```
Comment (surajeaton) [inline login-workflow/src/components/Error/ErrorManager.tsx:39]: This should be:
`sx?: SxProps<Theme>;`

Please update it in messageBoxConfig also.

File: login-workflow/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v4.0.2 (Unreleased)
+
+### Fixed
```
Comment (surajeaton) [inline login-workflow/CHANGELOG.md:10]: This should be moved to ### Added section.

File: login-workflow/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
...
+
+-   Add `sx` prop to Error Manager Dialog ([#521](https://github.com/etn-ccis/blui-react-workflows/issues/521)).
```
Comment (surajeaton) [inline login-workflow/CHANGELOG.md:12]: Sentence should be in the past tense `Added`.

**Review by surajeaton — APPROVED** [review]
Looks good.

**Pull request merged**